crates/subspace-farmer/src/archiving.rs

```diff
@@ -13,6 +13,11 @@ use tokio::{sync::oneshot, task::JoinHandle};
 
 const BEST_BLOCK_REQUEST_TIMEOUT: Duration = Duration::from_secs(60);
 
+pub struct ArchivedBlock {
```

Comment:
As mentioned in previous commit, this is not a block. And we don't need this data structure at all since we only used block number for printing logs, which we can do from the place where archiving happened.